hastexo/public/js/main.js

```diff
@@ -635,6 +446,9 @@ function HastexoXBlock(runtime, element, configuration) {
         terminal_client.disconnect()
 
         var dialog = $('#idle');
+        if (pause) {
+            dialog.find('.message').html('Your lab is currently active in a new window');
```

Comment:
Nit: I think calling it a "separate" window would be better. That's because the session may well be open for a period of, say, half and hour — at which point it's arguably not a "new" window anymore. :)